opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -297,9 +348,18 @@ impl UserEventsExporter {
                 }
                 eb.set_struct_field_count(cs_b_bookmark, cs_b_count);
 
-                eb.write(&log_es, None, None);
+                let result = eb.write(&log_es, None, None);
+                if result > 0 {
```

Comment:
if writing failed, then the exporter can return failure back to processor/provider, so upstream logs the failure.

Reply:
Done.